src/cloud_service.cpp

```diff
@@ -303,6 +296,34 @@ int CloudService::send_cb_wrapper(CloudServiceStatus status, JSONValue *rsp_root
     return rval;
 }
 
+void CloudService::publish_cb(
+    particle::Error error,
+    const char *event_name,
+    const char *event_data,
+    const void *event_context)
```

Comment:
If the `event_context` can only be interpreted as a `cloud_service_send_handler_t*` type then this is not really a context.

Reply:
It needs to have this type signature to be passed as a callback to `BackgroundPublish::publish`